storage/myisam/CMakeLists.txt

```diff
@@ -46,6 +46,12 @@ TARGET_LINK_LIBRARIES(myisamlog myisam)
 MYSQL_ADD_EXECUTABLE(myisampack myisampack.c COMPONENT Server)
 TARGET_LINK_LIBRARIES(myisampack myisam)
 
+IF(CMAKE_SYSTEM_NAME MATCHES AIX)
+  FOREACH(target myisam_ftdump myisamchk myisamlog myisampack)
+    TARGET_LINK_OPTIONS(${target} PRIVATE "-Wl,-berok")
```

Comment:
Frankly, since you add this berok to everything with very few exceptions, it is much better to have it in LINK_OPTIONS , or CMAKE_EXE_LINKER_FLAGS

Reply:
We do not want ``-berok`` for all executable (e.g. mariadbd, or client). I use it "only" to maria/myisam tools and unittests (was used to plugin, but this is corrected).

Reply:
OK, then what about something like
```
IF(CMAKE_SYSTEM_NAME MATCHES AIX)
  # Workaround linker bug on AIX
  SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-berok")
ENDIF()
```
at the start of both storage/myisam/CMakeLists.txt and storage/aria/CMakeLists.txt

smaller and better future-proof (w.r.t tools that can be added or removed)

Reply:
OK. Done.